spec/factory_specs/providers_spec.rb

```diff
@@ -7,9 +7,4 @@
     expect(provider).to be_instance_of(Provider)
     expect(provider).to be_valid
   end
-
-  it "creates a provider with enrichments" do
```

Comment:
why was the test removed?

Reply:
Sounds like it should be converted to a test of the `:with_enrichment` trait. Except that we tend not to test traits separately.